recipe/meta.yaml

```diff
@@ -73,19 +72,10 @@ outputs:
         - vispy >=0.14.1,<0.15.0a0
         - wrapt >=1.11.1
 
-        # additional dependencies for convenience in conda-forge
-        - fsspec
-        - pooch >=1.3.0
-        - zarr >=2.12.0
-
       run_constrained:
-        - napari-plugin-manager >=0.1.0a2,<0.3.0a0
         - pyside2 >=5.13.2,!=5.15.0
         - pyside6 <6.5|>=6.7
```

Comment:
thanks!

Reply:
Thanks for keeping things flexible. I don't really understand the plugin manager but cool!!!